aperture.go

```diff
@@ -209,6 +218,22 @@ func getConfig(configFile string) (*config, error) {
 	if cfg.ListenAddr == "" {
 		return nil, fmt.Errorf("missing listen address for server")
 	}
+
+	for _, service := range cfg.Services {
+		switch {
```

Comment:
Nice to have all the checks in one place now! Looking at it again, I think this `switch` (and the two related constants) should go into the loop of `prepareServices` in `proxy/service.go`. Sorry to request yet another code move... But the thing is, the log statement also doesn't work here because the log subsystem hasn't been initialized at this point.

Reply:
cool, good idea 👍